heartbeat/beater/manager.go

```diff
@@ -189,15 +200,42 @@ func (m *Monitor) Update(configs []*common.Config) error {
 
 	// start new and reconfigured tasks
 	for id, t := range all {
-		job := createJob(m.manager.client, t)
-		t.cancel = m.manager.jobControl.Add(t.config.Schedule, id, job)
+		processors, err := processors.New(t.config.Processors)
+		if err != nil {
+			logp.Critical("Fail to load monitor processors: %v", err)
```

Comment:
Not for this PR but thinking what we should do in this case? Alternatively we could return an error and move the creating of the jobs into a second for loop. Means all jobs only get created if all tasks work. Not sure what the user expects.